crates/rattler_conda_types/src/channel/mod.rs

```diff
@@ -66,13 +66,27 @@ impl Channel {
             Channel::from_url(url, platforms, config)
         } else if is_path(channel) {
             let path = PathBuf::from(channel);
-            let absolute_path = absolute_path(&path);
-            let url = Url::from_directory_path(absolute_path)
-                .map_err(|_| ParseChannelError::InvalidPath(path))?;
-            Self {
-                platforms,
-                base_url: url,
-                name: Some(channel.to_owned()),
+
+            #[cfg(not(target_arch = "wasm32"))]
+            {
+                let absolute_path = absolute_path(&path);
+                let url = Url::from_directory_path(absolute_path)
+                    .map_err(|_| ParseChannelError::InvalidPath(path))?;
+                Self {
+                    platforms,
+                    base_url: url,
+                    name: Some(channel.to_owned()),
+                }
+            }
+
+            #[cfg(target_arch = "wasm32")]
+            {
+                let url = Url::try_from("https://anaconda.org").unwrap();
```

Comment:
I don't like this :P

Is one of the functions above not supported?

Reply:
If thats the case, maybe we should just return an error here.

Reply:
The `Url::from_directory_path` doesn't exist on WASM (probably because there is no filesystem.

I am throwing an error instead.